src/assets/_project/_blocks/layout/location/qg-location.js

```diff
@@ -601,37 +601,37 @@ $(function () {
 
   // The user has allowed geolocation
   qgLocation.fn.processPositionData = function (response) {
-    var positionData = response['coords'];
+    var positionData = response.coords;
 
     qgLocation.fn.setPositionData(positionData);
     closeDropdown();
   };
 
   // The user has blocked geolocation
   qgLocation.fn.failure = function (response) {
-    var responseMessage = response['message'];
+    var responseMessage = response.message;
 
-    qgLocation['vars']['error_message'] = responseMessage;
+    qgLocation.vars.error_message = responseMessage;
   };
 
   // Save the position data to the browser
   qgLocation.fn.setPositionData = function (positionData) {
     var location = {
-      'latitude': positionData['latitude'],
-      'longitude': positionData['longitude'],
-      'locality': 'unknown',
+      latitude: positionData.latitude,
+      longitude: positionData.longitude,
+      locality: 'unknown',
     };
 
     // Save to cookie
     qgLocation.fn.saveLocationCookie(location);
 
     // Notify the rest of the page
-    $('body').trigger('custom', qgLocation['vars']['event_coordinates_set']);
+    $('body').trigger('custom', qgLocation.vars.event_coordinates_set);
   };
 
   // Save data to the location cookie
   qgLocation.fn.saveLocationCookie = function (cookieData) {
-    var cookieName = qgLocation['vars']['cookie_name'];
+    var cookieName = qgLocation.vars.cookie_name;
```

Comment:
I think they were using the array lookup instead of by object so that it does not crash if it does not exist.

Reply:
It used to create around 90 lint errors.
![image](https://github.com/qld-gov-au/qg-web-template/assets/126438691/cae79854-f3d7-4b91-91ef-3bbb2a35626c)
I created a PR to ignore this file from linting: https://github.com/qld-gov-au/qg-web-template/pull/1427